### PR TITLE
Don't show separator for single item (fixes #6593)

### LINKED ISSUE #6593: Extra "|" on pm popup
Not really a bug but a minor aesthetic issue.   PersonalMessages.template.php, template_pm_pop()

```
function template_pm_popup()
{
	global $context, $txt, $scripturl;

	// Unlike almost every other template, this is designed to be included into the HTML directly via $().load()
	echo '
		<div class="pm_bar">
			<div class="pm_sending block">
				', $context['can_send_pm'] ? '<a href="' . $scripturl . '?action=pm;sa=send">' . $txt['pm_new_short'] . '</a> | ' : '', '
				', $context['can_draft'] ? '<a href="' . $scripturl . '?action=pm;sa=showpmdrafts">' . $txt['pm_drafts_short'] . '</a>' : '', '
				<a href="', $scripturl, '?action=pm;sa=settings" class="floatright">', $txt['pm_settings_short'], '</a>
			</div>
```

if $context['can_draft'] is false, we're left with  "New | " on top.  Should be just "New".

### PR DESCRIPTION
**Pull request by Oldiesmann**
Signed-off-by: Michael Eshom <oldiesmann@gmail.com>

### CODE REVIEW

**Review by BrickOzp — APPROVED** [review]

File: Themes/default/PersonalMessage.template.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ function template_pm_popup()
 	echo '
 		<div class="pm_bar">
 			<div class="pm_sending block">
-				', $context['can_send_pm'] ? '<a href="' . $scripturl . '?action=pm;sa=send">' . $txt['pm_new_short'] . '</a> | ' : '', '
+				', $context['can_send_pm'] ? '<a href="' . $scripturl . '?action=pm;sa=send">' . $txt['pm_new_short'] . '</a>' . ($context['can_draft'] ? ' | ' : '') : '', '
```
Comment (jdarwood007) [inline Themes/default/PersonalMessage.template.php]: Why not combine with the below ternary?

Comment (Oldiesmann) [reply Themes/default/PersonalMessage.template.php]: For some reason I was thinking that $context['can_draft'] could be true when $context['can_send_pm'] is false. Will fix it shortly

Comment (jdarwood007) [reply Themes/default/PersonalMessage.template.php]: oh your right.  Logic does currently allow that.

**Review by live627 — APPROVED** [review]

**Comment by Oldiesmann** [comment]
Any particular reason this hasn't been merged in yet? I know it's rather trivial as far as PRs go but given its simplicity and the fact that it's already been approved I don't understand why it's been left untouched for over a month.

**Comment by jdarwood007** [comment]
Somebody just forgot.

**Pull request merged**